code/modules/keybindings/bindings_client.dm

```diff
@@ -70,9 +70,9 @@ GLOBAL_LIST_INIT(valid_keys, list(
 			break
 
 	if(holder)
-		holder.key_down(full_key, src)
+		holder.key_down(_key, src)
```

Comment:
This doesn't take into account control and stuff from what it looks like. Try doing thing like shift click for examine and check they still work

Reply:
I tested it again and stuff like examining, grabbing and unlocking things with the corresponding keys seem to work like normal.

Reply:
`key_down` is used only by this feature; no wonder why nothing broke by this sweeping change.